config/default.json5

```diff
@@ -24,21 +24,25 @@
     readyTimeout: 30000,
     checkInterval: 10000,
     disableSponsoredTx: false,
-
-    transferFeePercentage: 0 /* It's the fee that the fee collector will take from the execution of 
+    transferFeePercentage: 0  /* It's the fee that the fee collector will take from the execution of 
```

Comment:
I'd specify here that if the value is negative, the transferFeePercentage strategy isn't applyed

Reply:
And I'd use -1 as default value

Reply:
Agreed on put this on a separated task.

Reply:
I think we agreed on checking the values in a separate task, but we need to set it with a negative value not to apply this strategy

Reply:
Done.